src/components/SelectionList/Search/ActionCell.tsx

```diff
@@ -69,7 +73,7 @@ function ActionCell({
                     ]}
                     textStyles={StyleUtils.getFontSizeStyle(variables.fontSizeExtraSmall)}
                     iconStyles={styles.mr0}
-                    success
+                    success={shouldUseSuccessStyle}
```

Comment:
We should remove this condition. The badge should always be displayed with `success` style